apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -248,11 +248,12 @@ class ManageStudentsTable extends Component {
       true;
 
     return (
-      <span>
-        <Checkbox
-          disabled={disabled}
-        />
-      </span>
+      <ManageStudentsSharingCell
+        id={rowData.id}
+        isEditing={rowData.isEditing}
+        disabled={disabled}
+        checked={!rowData.sharingDisabled}
```

Comment:
`sharingDisabled={true}` is reflected as `checked={false}` but I think I account for that here, which is the opposite boolean issue you mentioned above?

Reply:
I wonder - not in this PR - how difficult/useful it would be to change `sharingDisabled` to `sharingEnabled` for sake of consistency?  If sharingEnabled is true, checked is true.  We'd have to update throughout the server-side code too, so probably a pain.

Reply:
It's actually fairly common to map the front end to a different name than the backend, and you could do that here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/manageStudents/manageStudentsRedux.js#L437
And then use the new name everywhere in JS. This is most common when ruby variables use `var_names_like_this` and javascript uses `variableNamesLikeThis`. I think that'd be a great thing to do.